### PR TITLE
Blog: TBD Innovators Oct 2024

### PR DESCRIPTION
**Pull request by taniashiba**
## What type of PR is this? (check all applicable)

- [x] 📖 Content

## Description

Blog post covering recent apps in our updated TBD Innovators showcase as of October 2024.

## Preview

https://deploy-preview-1860--tbd-website-developer.netlify.app/blog/tbd-innovators-oct24

### CODE REVIEW

File: site/blog/2024-10-29-tbd-innovators-1024.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  <meta property="twitter:domain" content="developer.tbd.website" />
+  <meta name="twitter:site" content="@tbdevs" />
+  <meta name="twitter:title" content="How Decentralized Apps Can Make Everyday Tasks Easy" />
+  <meta property="twitter:url" content='https://developer.tbd.website/blog/2024-10-29-tbd-innovators-1024' /> 
+  <meta name="twitter:description" content="Learn how TBD's Innovator apps using our technology and how they can impact you from today!" />
+  <meta name="twitter:image" content="https://developer.tbd.website/assets/images/blog-tbd-innovators-1024.png" />
+
+  <link rel="apple-touch-icon" href="https://developer.tbd.website/img/tbd-fav-icon-main.png" />
+</head>
+
```
Comment (EbonyLouis) [inline site/blog/2024-10-29-tbd-innovators-1024.md:27]: you're missing your thumbnail image

File: site/blog/2024-10-29-tbd-innovators-1024.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+<!--truncate-->
+
+As our open source projects continue to develop, our community members contribute to the global effort of decentralizing the web with their independent projects. Here are the latest contributions from them.
+
+## Ariton
+
+Developed by Sondre Bjellås (@sondreb), Ariton is a Web5 community SuperApp, a decentralized platform for building and managing communities! Ariton runs on any device with the ability to add any Mini Apps (or features) you want, like chat, groups, events, notes and more. Built on free and open standards, your identity and data is always in your full control. Currently in prototype stage, you can learn more and [try it out here](https://ariton.app/#learn).
+
+# Kin AI
```
Comment (EbonyLouis) [inline site/blog/2024-10-29-tbd-innovators-1024.md:40]: need consistency here: so two `#` instead of one

**Review by EbonyLouis — CHANGES_REQUESTED** [review]

File: site/blog/2024-10-29-tbd-innovators-1024.md (hunk 1 above)
Comment (taniashiba) [reply site/blog/2024-10-29-tbd-innovators-1024.md:28]: Added, thank you!

File: site/blog/2024-10-29-tbd-innovators-1024.md (hunk 2 above)
Comment (taniashiba) [reply site/blog/2024-10-29-tbd-innovators-1024.md]: Thank you, changed!

File: site/blog/2024-10-29-tbd-innovators-1024.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+Whenever I explore technology that's new to me, whether it be learning how decentralized apps work or what an open source tool does, it's seeing the technology in action that helps me understand whether or not it can impact me and my every day life. This is why every month at TBD, what better way to learn about our technology in action than from innovators using TBD's technologies today.
+
+<!--truncate-->
+
+As our open source projects continue to develop, our community members contribute to the global effort of decentralizing the web with their independent projects. Here are the latest contributions from them.
+
+## Ariton
+
+Developed by Sondre Bjellås (@sondreb), Ariton is a Web5 community SuperApp, a decentralized platform for building and managing communities! Ariton runs on any device with the ability to add any Mini Apps (or features) you want, like chat, groups, events, notes and more. Built on free and open standards, your identity and data is always in your full control. Currently in prototype stage, you can learn more and [try it out here](https://ariton.app/#learn).
```
Comment (blackgirlbytes) [inline site/blog/2024-10-29-tbd-innovators-1024.md:39]: i think the first sentence is actually a comma splice. If you want, you can split it up into two sentences. For example, "Developed by Sondre Bjellås (@sondreb), Ariton is a Web5 community SuperApp. It acts as a decentralized platform for building and managing communities"

Comment (taniashiba) [reply site/blog/2024-10-29-tbd-innovators-1024.md]: You're right! I felt weird about that first sentence, and that change just makes it better. Addressed!

**Review by EbonyLouis — APPROVED** [review]
🎉

**Review by blackgirlbytes — APPROVED** [review]

**Pull request merged**